tests/acceptance/reminder-list-test.js

```diff
@@ -56,10 +64,22 @@ test('adding new notes', function(assert) {
   })
 })
 
-test('add new note prompt', function(assert) {
-  visit('/');
+test('editing notes', function(assert) {
```

Comment:
Let's set a variable to whatever the title is first, and check that the edited title is different. Right now it just asserts that whatever you typed matches what you typed.